utils/manifest-utils/test-utils/Cargo.toml

```diff
@@ -14,4 +14,4 @@ tar = "0.4"
 openssl = "0.10"
 
 ya-manifest-utils = "0.2"
-ya-agreement-utils = "0.4.1"
```

Comment:
nitpick: probably this file does not need to change?

Reply:
New line at t he end removed.